db/models.py

```diff
@@ -1 +1,29 @@
 from django.db import models
+from django.db.models import (CharField, TextField,
+                              ForeignKey, DateTimeField, EmailField)
+
+
+class Race(models.Model):
+    name = CharField(max_length=255, unique=True)
+    description = TextField(blank=True)
+
+
+class Guild(models.Model):
+    name = CharField(max_length=255, unique=True)
+    description = TextField(null=True)
+
+
+class Skill(models.Model):
+    name = CharField(max_length=255, unique=True)
+    bonus = CharField(max_length=255)
+    race = ForeignKey(Race, on_delete=models.DO_NOTHING,)
```

Comment:
If you want to prevent deletion of a Race with associated Skill entries, consider PROTECT.